modules/kargoBidAdapter.js

```diff
@@ -95,6 +95,7 @@ function buildRequests(validBidRequests, bidderRequest) {
     },
     imp: impressions,
     user: getUserIds(tdidAdapter, bidderRequest.uspConsent, bidderRequest.gdprConsent, firstBidRequest.userIdAsEids, bidderRequest.gppConsent),
+    site: firstBidRequest.site
```

Comment:
`site` is actually nested in another object `ortb2`.
i.e: `firstBidRequest.ortb2.site`